### DISCUSSION TITLE
GE/Jasco ZW4003 14292 not in device catalog but works

### DISCUSSION THREAD

**Discussion by brantlk** [zwave-js/zwave-js#2452] [General] [answered]
I was able to pair a GE/Jasco ZW4003 14292 and operate it (turn the switch off and on). It's not in the Config DB Browser. There's a 14293 and other Jasco in-wall switches but not 14292. This is using home assistant.

**Comment by AlCalzone** [comment, marked as answer]
That is expected. The configuration files mostly are for UX purposes: predefined config parameters, device name and label, better association labels than the auto-discovered ones. To add a config for this one, please open an issue with the template for "missing config file" or (even better) provide a PR.
